sanic/mixins/exceptions.py

```diff
@@ -14,7 +14,7 @@ def _apply_exception_handler(self, handler: FutureException):
     def exception(
         self,
         *exceptions: Union[Type[Exception], List[Type[Exception]]],
-        apply: bool = True
+        apply: bool = True,
```

Comment:
It's a common convention nowadays to put a comma after last item too, when something is split over multiple lines. Makes it easier to add more when you don't have to touch the prior last line to add a comma. It also at least in some cases signals formatters (black and ruff) that this is to be kept split to avoid combining in one line even if possible (this feature is called magic comma).

There's also an option to remove the final comma, and I am not sure whether Ruff currently uses the magic comma with Sanic. I can dig into it in more detail if necessary.

Reply:
It is a fairly common to put comma even after the last item in expanded (multi line) format, so that adding more entries does not require adding a comma on the previous line. Black apparently also does this most of the time... And there appears to be no configuration for it.

This is one of the things that Ruff apparently intentionally deviates from Black in some yet not clearly specified cases. https://docs.astral.sh/ruff/formatter/black/#trailing-commas-are-inserted-when-expanding-a-function-definition-with-a-single-argument